docs/featurizers.rst

```diff
@@ -122,12 +122,6 @@ AtomCoordinates
 .. autoclass:: deepchem.feat.AtomicCoordinates
   :members:
 
-AdjacencyFingerprint
```

Comment:
Can you add `AdjacencyFingerprint` and `OneHotFeaturizer` back in? I'm concerned that we might have users using these featurizers so we shouldn't remove them

Reply:
> Can you add AdjacencyFingerprint and OneHotFeaturizer back in? I'm concerned that we might have users using these featurizers so we shouldn't remove them.

I didn't agree. First, `AdjacencyFingerprint` can not be used with any DeepChem models (The current docs also is wrong) and there are strange implementations (maybe, something wrong) like `get_atom_type`.  So, I think there is no one uses `AdjacencyFingerprint`. 

And then, `OneHotFeaturizer` also can not be used with any DeepChem models. If `OneHotFeaturizer` returns general one-hot vectors, I could agree that someone may use it. However, `OneHotFeaturizer` doesn't return general one-hot vectors. 

The shape of general one-hot vectors is `(sequence_length, one_hot_vector_length)` like this.
The case of "Can I eat the Pizza”
```
[[1. 0. 0. 0. 0.] #can
 [0. 0. 1. 0. 0.] #i
 [0. 1. 0. 0. 0.] #eat
 [0. 0. 0. 0. 1.] #the
 [0. 0. 0. 1. 0.]] #pizza
```

So, I think `OneHotFeaturizer` should return one-hot vectors like this.

The case of "CN=C=O”

```
charset = ['C', 'N', '=', 'O']
[[1. 0. 0. 0.] #C
 [0. 1. 0. 0.] #N
 [0. 0. 1. 0.] #=
 [1. 0. 0. 0.] #C
 [0. 0. 1. 0.] #=
 [0. 0. 0. 1.]] O
```

But, the return values of `OneHotFeaturizer` is really strange. The value has three dimensions. Many users can't understand how to use it. So, I think there is no one uses `OneHotFeaturizer`.

Reply:
Looking at it more, I agree we can remove `AdjacencyFingerprint` since I don't think anyone has been using it. For `OneHotFeaturizer` though, it seems that we need to fix it rather than remove it. Having a one-hot featurization for sequences is generally useful (https://github.com/deepchem/deepchem/issues/2150 suggests we need one that works for protein sequences). I suggest that we raise an issue documenting the needed fixes for `OneHotFeaturizer` and fixing it in a follow-up PR